TODO

```diff
@@ -34,9 +34,6 @@ Features:
   mounting a subdir of the root fs as actual root. This can be used as
   fstype-agnostic version of btrfs' rootflags=subvol=foobar.
 
-* add --copy-from and --copy-to command to systemd-dissect which copies stuff
-  in and out of a disk image
```

Comment:
Hmm, `--copy-{from,to}` are more self-explantory than `--extract/--add`. Esp. "add" is very generic... Do we have some "prior art" for the naming of this type of operations?

Reply:
I used "tar" as inspiration, i.e. "tar --extract" is the long form of "x" and "tar --add-file=FILE" is for adding in stuff...

Reply:
The "--copy-from=" and "--copy-to=" stuff I didn#t like to much since it wasn't clear what it copies from and to.